### DISCUSSION TITLE
deepflow-agent使用docker build 提示目录已存在

### DISCUSSION THREAD

**Discussion by guolifu** [deepflowio/deepflow#1455] [Q&A] [answered]
[ docker run --privileged --rm -it -v \
    $(pwd):/deepflow hub.deepflow.yunshan.net/public/rust-build bash -c \
    "source /opt/rh/devtoolset-8/enable && git clone https://github.com/deepflowys/deepflow.git /deepflow && cd /deepflow/agent && cargo build"
Unable to find image 'hub.deepflow.yunshan.net/public/rust-build:latest' locally
docker: Error response from daemon: Get "https://hub.deepflow.yunshan.net/v2/": net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers).
See 'docker run --help'.](fatal: destination path '/deepflow' already exists and is not an empty directory.)

**Comment by Nick-0314** [comment, marked as answer]
You should first confirm whether the pull image can be normal. You can clone the code locally (pay attention to the submodule), and then mount it into the container for compilation. You can add the enterprise wechat at the bottom of the readme to the communication group
